docs/browser-bindings/browser/README.md

```diff
@@ -1,6 +1,13 @@
+# @0x/mesh-browser - v9.1.0
```

Comment:
It doesn't look like `docs/browser-bindings/browser/reference.md` was created. We will need this for the gitbook setup to work correctly.

Reply:
I approved, but we should make sure this is addressed before merging